deps.edn

```diff
@@ -1 +1,17 @@
-{:deps {honeysql {:mvn/version "0.9.5" :exclusions [org.clojure/clojurescript]}}}
+{:deps    {honeysql/honeysql {:mvn/version "0.9.5" :exclusions [org.clojure/clojurescript]}}
+ :paths   ["src"]
+ :aliases {:test        {:extra-paths ["test"]}
+           :runner      {:extra-deps {com.cognitect/test-runner
+                                      {:git/url "https://github.com/cognitect-labs/test-runner"
+                                       :sha     "b6b3193fcc42659d7e46ecd1884a228993441182"}}
+                         :main-opts  ["-m" "cognitect.test-runner"
+                                      "-d" "test"]}
+           :cljs-runner {:extra-deps {olical/cljs-test-runner {:mvn/version "3.8.0"}}
+                         :main-opts  ["-m" "cljs-test-runner.main"]}
+           :jar         {:replace-deps {seancorfield/depstar {:mvn/version "2.0.165"}}
```

Comment:
What's the rationale for having both a :depstar alias and a :jar alias? Wouldn't the :depstar alias alone suffice for creating both uberjars and jars?

Reply:
Removed the `depstar` alias. I'm thinking, since this is a library, we shouldn't really need to build an uberjar. Thin jar should suffice.